### PR TITLE
85733 increase payments - missed change english

### PR DESCRIPTION
**Pull request by MarcoGoC**
### Description

List of proposed changes:
- english change and some nbsp added 

### What to test for/How to test
- Download and verify

### CODE REVIEW

File: i18n/api/en.ts [hunk 1]
```diff
@@ -338,19 +338,19 @@ const en: Translations = {
     },
     oasDeferralAvailable: {
       heading: 'You may be able to defer your payments',
-      text: 'To learn more about your option to delay your first Old Age Security pension payment, {LINK_OAS_DEFER_CLICK_HERE}.',
+      text: 'Learn more about the option to delay your first Old Age Security pension payment, {LINK_OAS_DEFER_CLICK_HERE}.',
```
Comment (alexunique0519) [inline i18n/api/en.ts]: the underline style should start from "option", also there is no "click here" in the description

File: i18n/api/en.ts [hunk 2]
```diff
@@ -338,19 +338,19 @@ const en: Translations = {
...
     },
     oasClawback: {
       heading: 'You may have to repay a part of your pension',
       text: 'Since {INCOME_SINGLE_OR_COMBINED} is over {OAS_RECOVERY_TAX_CUTOFF}, you may have to repay {OAS_CLAWBACK} in {LINK_RECOVERY_TAX}.',
     },
     oasIncreaseAt75: {
-      heading: 'Your payments will increase when you turn 75',
-      text: 'Once you turn 75, your payments will increase by 10%, meaning you will receive {OAS_75_AMOUNT} per month.',
+      heading: 'Your payments will increase when you turn&nbsp;75',
```
Comment (alexunique0519) [inline i18n/api/en.ts:348]: consider using a dangerouslySetInnerHTML for the span element inside of the CustomCollapse, as right now it shows "nbsp;" on the page

File: i18n/api/en.ts [hunk 3]
```diff
@@ -338,19 +338,19 @@ const en: Translations = {
...
+      text: 'Once you turn 75, your payments will increase by&nbsp;10%, meaning you will receive {OAS_75_AMOUNT} per month.',
     },
     oasIncreaseAt75Applied: {
-      heading: 'Your payments have increased because you are over 75',
-      text: 'Since you are over the age of 75, your payments have been increased by 10%.',
+      heading: 'Your payments have increased because you are over&nbsp;75',
```
Comment (alexunique0519) [inline i18n/api/en.ts:352]: consider using a dangerouslySetInnerHTML for the span element inside of the CustomCollapse, as right now it shows "nbsp;" on the page

**Review by alexunique0519 — DISMISSED** [review]

File: i18n/api/en.ts (hunk 2 above)
Comment (MarcoGoC) [reply i18n/api/en.ts:348]: done

**Review by alexunique0519 — APPROVED** [review]
Thanks

**Pull request merged**